src/SprykerEco/Zed/Unzer/UnzerConfig.php

```diff
@@ -104,13 +188,43 @@ public function isPaymentAuthorizeRequired(string $paymentMethodName): bool
     /**
      * @api
      *
-     * @param string $paymentMethodName
+     * @param string $paymentMethodKey
+     *
+     * @return string
+     */
+    public function getUnzerPaymentMethodKey(string $paymentMethodKey): string
+    {
+        foreach (static::UNZER_PAYMENT_METHOD_KEYS_MAP as $unzerPaymentMethodKey => $paymentMethodKeys) {
+            if (in_array($paymentMethodKey, $paymentMethodKeys)) {
+                return $unzerPaymentMethodKey;
+            }
+        }
+
+        throw new UnzerException('Unknown Unzer payment method key '.$paymentMethodKey.' detected.');
+    }
+
+    /**
+     * @api
+     *
+     * @param string $paymentMethodKey
      *
      * @return string
      */
-    public function getUnzerPaymentMethodKey(string $paymentMethodName): string
```

Comment:
This is also a part of our public API, so instead of removing old methods, deprecate them, please

Reply:
It's major release ...